pkg/roles/rolemanager/middleware/logging.go

```diff
@@ -23,6 +23,7 @@ type RoleManagerLoggingMiddleware struct {
 	logger  *slog.Logger
 }
 
+// NewRoleManagerLoggingMiddleware adds logging facilities to the core service.
```

Comment:
Use the same approach: `NewLogging` (and `NewAuth` snd `NewMetric`) here, so we have something like `middlewarealias.NewLogging...`.